src/tools/gophish_complete.py

```diff
@@ -46,46 +48,66 @@
 requests.packages.urllib3.disable_warnings()
 
 
-def get_campaigns(api):
-    """Return a dictionary containing all campaigns."""
+def get_campaign_id(campaign_name, campaigns):
+    """Get campaign id from campaign name.
+
+    Args:
+        campaign_name (string): Full campaign name.
+        campaigns (dict): Campaign id as key, campaign name as value.
+
+    Raises:
+        LookupError: When the campaign name is not found, raise exception.
+
+    Return:
+        Return campign id that corresponds to campaign name provided.
+    """
+    for campaign_id, name_value in campaigns.items():
+        if name_value == campaign_name:
+            return campaign_id
+
+    raise LookupError(f'Campaign Name "{campaign_name}" not found.')
+
+
+def get_campaigns(api, assessment_id):
+    """Return a dictionary containing all campaigns.
+
+    Args:
+        api (GoPhish API): Connection to GoPhish server via the API
+        assessment_id (string): Assessment identifier to get campaigns from.
+
+    Raises:
+        LookupError: If no campaigns are found for the assessment identifier, raise exception.
```

Comment:
`LookupError: No campaigns found for the provided assessment id.`

Reply:
Fixed in commit 1ab0c81.